tests/E2E/test/campaigns/high/13_employee/1_employee.js

```diff
@@ -24,18 +25,25 @@ scenario('Create employee', client => {
   test('should set "Password" input', () => client.waitAndSetValue(Employee.password_input, '123456789'));
   test('should choose "Permission profile" option', () => client.waitAndSelectByValue(Employee.profile_select, '4'));
   test('should click on "Save" button', () => client.waitForExistAndClick(Employee.save_button));
+  test('should verify the appearance of the green validation', () => client.checkTextValue(Employee.success_panel, 'Successful creation.'));
 }, 'common_client');
 
 scenario('Check the employee creation', client => {
   test('should search the created employee', () => client.waitAndSetValue(Employee.email_search_input, 'demo' + date_time + '@prestashop.com'));
   test('should click on "Search" button', () => client.waitForExistAndClick(Employee.search_button_team));
-  test('should check the result', () => client.checkTextValue(Employee.search_result, "1"));
+  test('should check the result', () => {
+    return promise
+      .then(() => client.waitForVisible(Employee.search_result))
+      .then(() => client.getTextInVar(Employee.search_result,'employee_number'))
+      .then(() => tab['employee_number'] = /\(([^)]+)\)/.exec(tab['employee_number'])[1])
```

Comment:
use global.tab instead of tab everywhere please.
Otherwise don't declare the variable as global :thinking:

Reply:
done [`b2fe946`](https://github.com/PrestaShop/PrestaShop/pull/13545/commits/b2fe946f33029474cfc9869481ef87377d5a4d33)